.github/workflows/susemanager-sls-compatible-python26-syntax.yml

```diff
@@ -0,0 +1,40 @@
+name: Check for non-compatible Python 2.6 in susemanager-sls Salt modules
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened]
+    paths: 'susemanager-utils/susemanager-sls/src/**/*'
+
+jobs:
+  susemanager_sls_check_non_python26_syntax:
+    runs-on: ubuntu-latest
+    container: centos:6
+    steps:
+      - name: Check out repository code
+        uses: actions/checkout@v1
```

Comment:
Is there a reason we don't use the current version?

Reply:
I had to use `v1` as it was the only version compatible with Python 2.6.

Reply:
Then we should add a comment, otherwise we'll forget why we use the old version the next time we look at the workflow yaml.

Reply:
Added a comment to clarify this :+1: